### ISSUE TITLE
Icon request: Pan

### ISSUE THREAD

**Issue by derrideanauthor** [Templarian/MaterialDesign#2799]
I think a great addition to the set would be a grab-and-drag cursor similar to photoshop's hand cursor which shows when you pan the canvas around;

**Comment by PeterShaggyNoble** [comment]
Would `cursor-move` suit your needs in the meantime?

[![cursor-move](https://user-images.githubusercontent.com/15157491/36024087-834ab0ae-0d86-11e8-86d8-eb682db93d06.png)](https://materialdesignicons.com/icon/cursor-move)

**Comment by derrideanauthor** [comment]
Yes I am actually using this one for now. Though it communicates movement of an object within a context and not the context itself - for which a hand-grab cursor would communicate well :)

**Comment by xaviergonz** [comment]
I was about to make the very same request :D

**Comment by augustin32** [comment]
There is already an icon for this in Google's official set
https://material.io/icons/#ic_pan_tool

**Comment by PeterShaggyNoble** [comment]
@augustin32, that's already being tracked in #1529. Leaving this one open as a request for the closed hand.
